plane/source/docs/battery-voltage-compensation.rst

```diff
@@ -0,0 +1,38 @@
+.. _battery-voltage-compensation:
+
+
+============================
+Battery Voltage Compensation
+============================
+
+As a flight progresses the battery voltage will gradually lower as power is consumed. This affects the maximum climb rate, and the cruising speed in systems with no airpeed sensor. Plane provides two mechanisms to cope with this, one for forward motor(s) and another for VTOL motors.
+
+Forward Motor Compensation
+==========================
+
+This is provided using:
+
+- :ref:`FWD_BAT_VOLT_MAX<FWD_BAT_VOLT_MAX>` : This sets the voltage from which lower voltages will result in compensating the actual applied forward throttle upwards. A value of "0", disables this function. Setting this to 4.2V * number of cells is a good value.
+- :ref:`FWD_BAT_VOLT_MIN<FWD_BAT_VOLT_MIN>` : Sets the lower limit at which throttle compensation will be done to prevent dragging the battery down excessively due to increased throttle scaling. Setting this to 3.3V * number of cells is a good value.
+- :ref:`FWD_BAT_IDX<FWD_BAT_IDX>` : Determines if the first or second battery monitor voltage is used.
```

Comment:
we can have more than two battery monitors (up to 10). So needs to be "Determines which battery index is used, with zero for the first battery".